hips/draw/simple.py

```diff
@@ -1,76 +1,89 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
-from typing import Generator, Any
 import numpy as np
+from typing import Tuple
 from astropy.wcs.utils import proj_plane_pixel_scales
 from skimage.transform import ProjectiveTransform, warp
 from ..tiles import HipsSurveyProperties, HipsTile, HipsTileMeta
-from ..utils import WCSGeometry, compute_healpix_pixel_indices
+from ..utils import WCSGeometry, compute_healpix_pixel_indices, get_hips_order_for_resolution
+
 
 __all__ = [
-    'draw_sky_image',
     'make_sky_image',
     'SimpleTilePainter',
-    'compute_matching_hips_order'
+
 ]
 
 __doctest_skip__ = [
-    'compute_matching_hips_order',
+    'SimpleTilePainter'
 ]
 
 
-# TODO: Fix type annotation issue
-def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any],
-                   hips_survey: HipsSurveyProperties, tile_format: str) -> np.ndarray:
-    """Draw sky image using the simple and quick method.
+class SimpleTilePainter:
+    """Paint HiPS tiles onto an all-sky image using a simple projective
```

Comment:
Docstring rule: Always have a one-line summary docstring, optionally followed by an empty line and more text.

In this case, the docstring we had on `draw_sky_image` would be good for the `SimpleTilePainter` class IMO:
```
Draw sky image using the simple and quick method.
```